lib/ui/instrumentpanel.js

```diff
@@ -263,14 +291,15 @@ InstrumentPanel.prototype.onWidgetChange = function(widget) {
 
 
 InstrumentPanel.prototype.persist = function() {
+  const skClientOptions = this.SignalkClient.options;
```

Comment:
Here we are reaching into SignalkClient and assume that there is a property called `options`. I don't think that is part of the client's public api and may change without notice, so I think it is a bit bad practise and brittle. 

We should either hold on to the parameters ourselves or add to skClient's api something like `getOptions`.

Reply:
fixed by commit: 2df0c4a

Reply:
Oh. I am not very familiar with the SK client, but it looks to me like `client.get(path)` fetches data from the server's http api. That is not what I was after and I assume that it does not work here the way accessing client.options properties would? 

I meant that if we use `client.options` object's properties directly @fabdrol may decide to move things around inside the client's code and the object or some of its' properties may be renamed and IP will break, since Fabian is not aware the somebody is using the client's internals in this way.

Instead I proposed to add a `getOptions()` function to the client, so that it is obvious that the object is exposed and later sk client development will consider it as part of the client's API.

Reply:
Oh, I did not realise that client.get('propertyName') is a supported pattern. Then we can go with this?

Reply:
Yes, we can ;-)